crates/uv-client/src/base_client.rs

```diff
@@ -376,11 +378,42 @@ impl<'a> BaseClientBuilder<'a> {
             path_exists
         });
 
+        // Checks for the presence of `SSL_CERT_DIR`.
+        // Certificate loading support is delegated to `rustls-native-certs`.
+        // See https://github.com/rustls/rustls-native-certs/blob/813790a297ad4399efe70a8e5264ca1b420acbec/src/lib.rs#L118-L125
+        let ssl_cert_dir_exists = env::var_os(EnvVars::SSL_CERT_DIR).is_some_and(|dirs| {
+            // Read `SSL_CERT_DIR`, split it on the platform delimiter (`:` on Unix, `;` on Windows),
+            // and return the existing/missing entries as `PathBuf`s.
+            let (existing, missing): (Vec<_>, Vec<_>) =
+                env::split_paths(&dirs).partition(|p| p.exists());
+
+            if existing.is_empty() {
+                warn_user_once!("Ignoring invalid `SSL_CERT_DIR`. None of the entries exists.");
```

Comment:
```suggestion
                warn_user_once!("Ignoring invalid `SSL_CERT_DIR`. None of the entries exist.");
```

I would also split between "None of the entries exist" and "The directory does not exist" depending on the length of `missing`.